### DISCUSSION TITLE
Is there a way to get a link to the snapshot file?

### DISCUSSION THREAD

**Discussion by kmanan** [tsightler/ring-mqtt#933] [Support Q&A]
Hi, I'm trying to send the snapshot to Ollama via Vision LLM. I tried https://domain/<entity_picture value>?< < access token > and that returns 403 forbidden. 

I would ideally prefer not storing the images separately, just take the last taken snapshot URL

**Comment by tsightler** [comment]
What URL do you want?  You can access the snapshot camera in HA using the same method as any other camera in HA, but you will need to authenticate for external access.  Currently the URL for the thumbnail from Ring isn't sent via ring-mqtt, but it would be an easy add.
